frontend/src/features/mapSideBar/MapSideBarContainer.tsx

```diff
@@ -25,15 +27,28 @@ export const MotiInventoryContainer: React.FunctionComponent = () => {
   const [ltsaData, setLtsaData] = useState<LtsaOrders | undefined>(undefined);
   const [apiProperty, setApiProperty] = useState<IPropertyApiModel | undefined>(undefined);
   const [ltsaDataRequestedOn, setLtsaDataRequestedOn] = useState<Date | undefined>(undefined);
+  const [showPropertyInfoTab, setShowPropertyInfoTab] = useState(true);
 
   // First, fetch property information from PSP API
   const { getPropertyWithPid } = useProperties();
   useEffect(() => {
     const func = async () => {
-      if (!!pid) {
-        const propInfo = await getPropertyWithPid(pid);
-        if (isMounted() && propInfo.pid === pidFormatter(pid)) {
-          setApiProperty(propInfo);
+      try {
+        if (!!pid) {
+          const propInfo = await getPropertyWithPid(pid);
+          if (isMounted() && propInfo.pid === pidFormatter(pid)) {
+            setApiProperty(propInfo);
+            setShowPropertyInfoTab(true);
+          }
+        }
+      } catch (e) {
+        // PSP-2919 Hide the property info tab for non-inventory properties
+        // We get an error because PID is not on our database
+        if (axios.isAxiosError(e)) {
+          const axiosError = e as AxiosError<IApiError>;
+          if (axiosError?.response?.status === 400) {
```

Comment:
darn, does the api really return a 400 and not a 404 in this case?

Reply:
eech, yep, it does. Should we return a 404 from here instead? 
            else if (ex is KeyNotFoundException)
            {
                code = HttpStatusCode.BadRequest;
                message = "Item does not exist.";

                _logger.LogDebug(ex, "Middleware caught unhandled exception.");
            }

Reply:
Sadly it does. I was hoping for a 404 but we get 400. Might want to file a tech debt ticket to see if we can change it across the app for that keynotfound exception

Reply:
I can fix it there. Good eye